build/images/Dockerfile.build.coverage

```diff
@@ -24,7 +24,9 @@ RUN go mod download
 
 COPY . /antrea
 
-RUN make antrea-agent antrea-controller antrea-cni antctl-linux antrea-controller-instr-binary antrea-agent-instr-binary antctl-instr-binary
+# To make the GO binaries portable, so that they can run on OS,
+# other from the one on which it was built we need to disable CGO.
+RUN CGO_ENABLED=0 make antrea-agent antrea-controller antrea-cni antctl-linux antrea-controller-instr-binary antrea-agent-instr-binary antctl-instr-binary
```

Comment:
only `antctl-linux` and `antctl-instr-binary` should be built with CGO disabled, as I pointed out in my review of your other PR